libraries/ms-helper/inc/exported_enums.h

```diff
@@ -65,3 +65,12 @@ typedef enum EERelayState {
   EE_RELAY_STATE_CLOSE,
   NUM_EE_RELAY_STATES,
 } EERelayState;
+
+typedef enum EESteeringEvents {
+  EE_STEERING_INPUT_HORN,
+  EE_STEERING_RADIO_PPT,
+  EE_STEERING_HIGH_BEAM_FORWARD,
+  EE_STEERING_HIGH_BEAM_REAR,
+  EE_STEERING_REGEN_BRAKE,
+  EE_STEERING_INPUT_CC_TOGGLE_PRESSED
```

Comment:
remove this entire enum.

Reply:
This shouldn't be in exported enums, it should be in `steering_events.h` and should be part of your project.